src/stories/Library/content-list/ContentList.tsx

```diff
@@ -0,0 +1,40 @@
+import clsx from "clsx";
+import {
+  ContentListItem,
+  ContentListItemProps,
+} from "../content-list-item/ContentListItem";
+import ContentListItemStacked from "../content-list-item/ContentListItemStacked";
+
+type ContentListProps = {
+  items: ContentListItemProps[];
+};
+
+const ContentList: React.FC<ContentListProps> = ({ items }) => {
+  return (
+    <ul className="content-list">
+      {items.map((item, index) => {
+        // Check if current event should be stacked with the previous one
+        const isStacked =
+          items[index].eventSeriesId === items[index - 1]?.eventSeriesId ??
```

Comment:
This references something event specific in what is supposed to be a generic component.

I am not too happy about this. An alternate suggestion would be to just add a `stacked` attribute to each entry in the sample data and base the stacking on this.